src/org/parosproxy/paros/core/scanner/VariantHeader.java

```diff
@@ -33,14 +35,19 @@
  */
 public class VariantHeader implements Variant {
 
-    // I've found an XSS using this payload on the Host header
-    // "%s-->\">'>'\"<sfi%06uv%06u>"
-
-    public static final String[] injectableHeaders = {
-        HttpRequestHeader.USER_AGENT,
-        HttpRequestHeader.REFERER,
-        HttpRequestHeader.HOST
+    public static final String[] nonInjectableHeaders = {

```

Comment:
Consider making it private (it's mutable). If we really need to expose the headers it's better with an unmodifiable `Collection`.